bootstrap/terraform/aws-bootstrap/aws-lb-controller.tf

```diff
@@ -196,6 +214,28 @@ resource "aws_iam_policy" "alb" {
                 }
             }
         },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "elasticloadbalancing:AddTags"
+            ],
+            "Resource": [
+                "arn:aws:elasticloadbalancing:*:*:targetgroup/*/*",
+                "arn:aws:elasticloadbalancing:*:*:loadbalancer/net/*/*",
+                "arn:aws:elasticloadbalancing:*:*:loadbalancer/app/*/*"
+            ],
+            "Condition": {
```

Comment:
Same here, I'm concerned this has the same problems w/ borked iam perms

Reply:
This is the new upstream IAM permission that includes the fix for the tags. I just tested with creating a LoadBalancer service and it worked as expected.

Reply:
ok awesome